src/Etterna/Globals/StepMania.cpp

```diff
@@ -3,11 +3,13 @@
 #include "Etterna/Globals/rngthing.h"
 
 // Rage global classes
+#include "Core/Services/Locator.hpp"
+#include "Core/Misc/PlogLogger.hpp"
 #include "Etterna/Singletons/GameSoundManager.h"
 #include "Etterna/Models/Misc/LocalizedString.h"
 #include "RageUtil/Graphics/RageDisplay.h"
 #include "RageUtil/Misc/RageInput.h"
-#include "RageUtil/Misc/RageLog.h"
+#include "RageUtil/Misc/RageMath.h"
```

Comment:
What's this `#include` for?

Reply:
I think git is detecting that as removing one and adding another, but only because when I merged, the RageMath.h was added on one side of the merge, and RageLog was removed on the other side. So it looks like a replacement, which it kind of is, but only because it was during the merge.